### PR TITLE
Improve OBF-GPX points appearance, add route=segment with relation_gpx=yes

### PR DESCRIPTION
**Pull request by RZR-UA**

### CODE REVIEW

File: obf_creation/rendering_types.xml [hunk 1]
```diff
@@ -1170,11 +1170,21 @@
 		<type tag="start_ele" additional="text" minzoom="3"/>
 		<type tag="ele_graph" additional="text" minzoom="3"/>
 
-		<type tag="width" additional="true" minzoom="3"/>
```
Comment (xmd5a2) [inline obf_creation/rendering_types.xml:1173]: Why remove **width**?

Comment (RZR-UA) [reply obf_creation/rendering_types.xml:1173]: https://github.com/osmandapp/OsmAnd-resources/pull/1139/commits/6528f37b261e0e267bdb4b08340e4ba0b210cd1d

It's used to display width=thin|medium|bold of user gpx tracks.
Now replaced with gpx_width as well as most other gpx_ attributes.

Comment (RZR-UA) [reply obf_creation/rendering_types.xml:1173]: resolved with xmd5a

File: poi/poi_types.xml [hunk 2]
```diff
@@ -5320,6 +5320,14 @@
 				</poi_additional_category>
 			</poi_type>
 			<poi_type name="route_article" tag="route_type" value="article"/>
+
+      <!-- OBF-GPX relation route types https://wiki.openstreetmap.org/wiki/Relation:route#Route_types_(route) -->
+			<poi_type name="osm_hiking" tag="route_type" value="hiking"/>
```
Comment (vshcherb) [inline poi/poi_types.xml:5325]: Rename route_hiking

File: poi/poi_types.xml [hunk 3]
```diff
@@ -5375,12 +5383,17 @@
...
-				<!-- OBF-GPX point icon extensions -->
+				<!-- OBF-GPX point: icon extensions -->
 				<poi_additional name="gpx_bg" tag="gpx_bg" type="text"/>
 				<poi_additional name="gpx_icon" tag="gpx_icon" type="text"/>
 				<poi_additional name="points_groups_category" tag="points_groups_category" type="text"/> <!-- GPX <type> -->
 			</poi_type>
 			<poi_additional name="color" tag="colour" type="text"/> <!-- global for "routes" category -->
+			<!-- OBF-GPX track/points: OSM tags (prefixed with osm_) -->
+			<poi_additional name="osm_id" tag="osm_id" type="text"/>
+			<poi_additional name="osm_ref" tag="osm_ref" type="text"/>
```
Comment (vshcherb) [inline poi/poi_types.xml:5394]: Just ref

File: poi/poi_types.xml [hunk 4]
```diff
@@ -5375,12 +5383,17 @@
...
+			<poi_additional name="osm_name" tag="osm_name" type="text"/>
```
Comment (vshcherb) [inline poi/poi_types.xml:5395]: Not needed just name

**Pull request merged**